test/e2etest/util/util.go

```diff
@@ -0,0 +1,129 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/gardener/test-infra/pkg/util"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func DownloadFile(url, dir string) (filePath string, err error) {
+	request, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+	fileName := path.Base(request.URL.Path)
+	filePath = filepath.Join(dir, fileName)
+	fileData, err := util.DownloadFile(&http.Client{}, url)
+	if err != nil {
+		return "", err
+	}
+	if err := ioutil.WriteFile(filePath, fileData, 0777); err != nil {
+		return "", err
+	}
+	return filePath, nil
+}
+
+func UniqueStrings(stringSlice []string) []string {
+	keys := make(map[string]bool)
+	list := []string{}
+	for _, entry := range stringSlice {
+		if _, value := keys[entry]; !value {
+			keys[entry] = true
+			list = append(list, entry)
+		}
+	}
+	return list
+}
+
+func GetEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+func RunCmd(command, execPath string) error {
```

Comment:
maybe we should also let the stdout and stderr parameter customizable.

Then this functions could be reused for all commands

Reply:
Sure, let's discuss in person how the interface should look like